### PR TITLE
Oracle8 build

### PR DESCRIPTION
**Pull request by JasonMarechal25**
Add new workflow to build on oracle 8 platform.

### CODE REVIEW

File: .github/workflows/build_oracle8.yml
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml]:
```suggestion
          
```
it's added also in requirements.txt. why do we need this package?

Comment (JasonMarechal25) [reply .github/workflows/build_oracle8.yml]: For some reason it is required (I think by pyinstaller) to be installed and can't be installed by requirements

**Review by a-zakir — COMMENTED** [review]
i'm ok with this. the branch has to be updated with develop before merge.

**Comment by JasonMarechal25** [comment]
Need to look into some CI failure

**Comment by JasonMarechal25** [comment]
Blocked by https://github.com/AntaresSimulatorTeam/Antares_Simulator/issues/1741

File: .github/workflows/build_oracle8.yml
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml]:
```suggestion
```

File: .github/workflows/build_oracle8.yml
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml:67]: what's the purpose of python3-devel? later on line 86 you installed python3.

Comment (JasonMarechal25) [reply .github/workflows/build_oracle8.yml:67]: devel version is required to build wheel which is required to pyinstaller

**Review by a-zakir — APPROVED** [review]

**Comment by a-zakir** [comment]
re-run checks before merge

**Pull request merged**
